### PR TITLE
There was errors placing ships in the project 4 code that did not exist in project 3

### PR DESCRIPTION
**Pull request by kiefk**
This is a copy of my project three code.

### CODE REVIEW

**Comment by kiefk** [comment]
This now also have the new Easy, Hard, and Mode classes. They are currently empty

**Comment by narxy** [comment]
github claims no files were changed

**Comment by RyanRapp14** [comment]
Ok I don't see the new files. Did you try uploading them?

**Comment by RyanRapp14** [comment]
These look good. I can work with these

**Comment by kiefk** [comment]
Looks like it was trying to push to a different repository so it should be fixed now

File: src/main/java/edu/oregonstate/cs361/battleship/Easy.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package edu.oregonstate.cs361.battleship;
+
+/**
+ * Created by Keana on 3/15/2017.
```
Comment (narxy) [inline src/main/java/edu/oregonstate/cs361/battleship/Easy.java:4]: :thinking:

**Pull request merged**